### DISCUSSION TITLE
How to get the OSM Id of an object

### DISCUSSION THREAD

**Discussion by lefuturiste** [osmandapp/OsmAnd#16703] [Q&A] [answered]
There are many use case, for example if you want to see the history of an POI or a way. You want to open the webpage https://www.openstreetmap.org/node/1000000001/history to browse the history of that object.
or you may want to just copy the ID.
How can I do that?

**Comment by lefuturiste** [comment, marked as answer]
From a telegram conversation
> Enable the OSM editing plugin. Then tap on an object. It will show you the OSM URL to this element (at least for many objects, doesn't work for all of them)

**Reply by Max1234-Ita** [reply]
I tried this on my device.
The OSM plugin is enabled, but when I tap on whatever object in the map nothing is shown.

The only information I can get is the URL of the selected location after long-tapping and selecting Details option (i.e. https://osmand.net/map?pin=44.99004,9.00794#18/44.99004/9.00794), but that's a position in the web map, not the OSM id of the object.

Can you post a short video demonstrating the feature, please? It would be very appreciated!
